src/schema/objects/entities.yaml

```diff
@@ -201,7 +201,7 @@ reconstruction:
   entity: rec
   description: |
     The `rec-<label>` key/value can be used to distinguish
-    different reconstruction algorithms (for example ones using motion
+    different reconstruction algorithms (for example `MoCo` for the ones using motion
```

Comment:
This feels like common enough occurrence in scanners output to add it by name as an example